Post/views.py

```diff
@@ -11,8 +12,20 @@ class PostListView(ListView):
     context_object_name = 'posts'
 
 
+@login_required
+def AddAPost(request):
```

Comment:
maybe AddPost is a bit more clear

Reply:
I changed it to CreateNewPost, is it ok? @maayanhd